### ISSUE TITLE
Recuperar funcionalidad de "mensajes no leídos"

### ISSUE THREAD

**Issue by deivid-rodriguez** [alabs/nolotiro.org#299]
La anterior versión de la mensajería de nolotiro incluía la posibilidad de ver si los mensajes habían sido leídos por el destinatario. Fue intencional quitarlo? Creo que sería útil recuperarlo para la nueva versión de la mensajería.

**Comment by andreslucena** [comment]
No, no fue intencional, creo que se me paso hacerlo en su momento.

**Comment by deivid-rodriguez** [comment]
:+1:
